CHANGELOG.rst

```diff
@@ -10,6 +10,26 @@ Changelog
 v.2.9.0 TBA
 ==================
 
+ * This version requires script 'migrate_package_activity.py' to be run
```

Comment:
would it be possible to detect un-migrated activities and display them with a <needs migration> message instead of requiring the site to be offline while the migration is happening?

Reply:
The migration is designed to be run while the site is online. I've just re-worded this to make that clear.

Hopefully the site admin would practise a CKAN upgrade before doing it on live. They would get the [error message](https://github.com/ckan/ckan/blob/d253130526ea538e1fa49229e91f1d19d8711c6c/ckan/model/__init__.py#L284) when they did 'paster db upgrade' telling them to the this migration first, as explained [here](https://github.com/ckan/ckan/wiki/Migrate-package-activity#if-you-tried-to-upgrade-from-ckan-28-or-earlier-to-ckan-29-and-it-stopped-at-paster-db-upgrade). Hopefully that's clear.